product_docs/docs/eprs/7/03_installation/installing_jdbc_driver.mdx

```diff
@@ -7,15 +7,10 @@ using migrating to or from EDB Postgres Advanced Server, use the EDB JDBC driver
 
```

Comment:
I think the having the directions on when and how to get and install the EDB JDBC driver in paragraph form and then have the info for the other drivers be in bulleted form is confusing.  This mix of styles caused me to overlook the important details in the opening paragraph then have to reread when I didn't see the EDB JDBC driver listed like the other drivers.

Also, the first sentence has this phrase which needs some rewording:

> you are using: using migrating

Reply:
Overall the formatting changes look great